app/setup.py

```diff
@@ -324,6 +327,17 @@ def setup_submitter(application):
         raise Exception("Unknown EQ_SUBMISSION_BACKEND")
 
 
+def setup_publisher(application):
+    if application.config["EQ_PUBLISHER_BACKEND"] == "pubsub":
+        application.eq["publisher"] = PubSubPublisher()
+
+    elif application.config["EQ_PUBLISHER_BACKEND"] == "log":
+        application.eq["publisher"] = LogPublisher()
+
+    else:
+        raise Exception("Unknown EQ_PUBLISHER_BACKEND")
```

Comment:
Why the base Exception?

Reply:
Just because that is what we currently do, but ya it could be more specific but I would then suggest updating all existing ones.

Reply:
When you say all existing ones, do you mean across Runner?

Reply:
Well certainly within this file, as our pattern currently is to raise generic exception when env vars are messing/not as expected..